src/Utilities/MuxLogger.cs

```diff
@@ -651,7 +651,6 @@ private void RaiseMessageEvent(object sender, BuildMessageEventArgs buildEvent)
                             // first unregister all loggers, since other loggers may receive remaining events in unexpected orderings
                             // if a fellow logger is throwing in an event handler.
                             UnregisterAllEventHandlers();
-
```

Comment:
Only a slight preference, but I do have a stronger preference for either whitespace or no whitespace rather than mixed—the catch (LoggerException) right above this has no space, for instance.